docs/pull-clickhouse-docs-hook.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+# The script to update user-guides documentation repo
+# https://github.com/ClickHouse/clickhouse-docs
+
+WORKDIR=$(dirname "$0")
+WORKDIR=$(readlink -f "${WORKDIR}")
+cd "$WORKDIR"
+
+UPDATE_PERIOD_HOURS="${1:-24}"  # By default update once per 24 hours; 0 means "always update"
+
+if [ ! -d "clickhouse-docs" ]; then
+    echo "There's no clickhouse-docs/ dir, run get-clickhouse-docs.sh first to clone the repo"
+    exit 1
```

Comment:
```
git -C nonexisting pull
fatal: cannot change to 'nonexisting': No such file or directory
```

Reply:
So, I agree to install the hook

```
> ./get-clickhouse-docs.sh
Enter the URL type (ssh | https): ssh
Would you like to setup git hook for automatic update? (y|n): y
Appending '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/docs/pull-clickhouse-docs-hook.sh 24' to /home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/.git/hooks/post-checkout
Cloning into 'clickhouse-docs'...
remote: Enumerating objects: 19561, done.
remote: Counting objects: 100% (19561/19561), done.
remote: Compressing objects: 100% (7644/7644), done.
^Cceiving objects:   1% (196/19561)
```

And canceling it in the process

Then checkout/pull is broken

```
> git checkout master
Switched to branch 'master'
Your branch is behind 'origin/master' by 81 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
.git/hooks/post-checkout: line 1: /home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/docs/pull-clickhouse-docs-hook.sh: No such file or directory
> git checkout -
Switched to branch 'better_script_for_updating_docs'
Your branch is up to date with 'origin/better_script_for_updating_docs'.
There's no clickhouse-docs/ dir, run get-clickhouse-docs.sh first to clone the repo
```

Reply:
Sounds like "If I shoot my leg it hurts". You can easily fix it by:
 - Running `./docs/get-clickhouse-docs.sh` once again
 - Clonning the missing `clickhouse-docs` manually
 - Removing `.git/hooks/post-checkout` completely or removing the line from it
 - Adding `||:` to the line in `.git/hooks/post-checkout`, so it will exit with code 0

And the only "broken" thing is that it exits with code 1, checkout/pull works as expected

Reply:
The action of adding the hook should go after the pull succeed, than it won't break

Reply:
It still will "break" if you checkout an old branch that does not contain the script. Or if you manually remove `clickhouse-docs`. Or if you do some weird stuff. 

Let's add `||:` by default